recipes/pyflann/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.8.4" %}
+
+package:
+    name: pyflann
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/mariusmuja/flann.git
+    git_tag: 40d63fc7db5d1f6dd67364f24e4579aedb57e7b6
+
+build:
+    number: 0
+    string: py{{CONDA_PY}}_{{PKG_BUILDNUM}}_g{{GIT_FULL_HASH[:7]}}
+
+requirements:
+    build:
+        - python {{PY_VER}}*
```

Comment:
That is not need in conda-forge as far as I know. But @pelson can comment on that.